src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -1,25 +1,1002 @@
+                        Inventory_DeleteItem(value, slot);
+                    } else if (slot == SLOT_TRADE_DEED) {
+                        if (CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                            if (INV_CONTENT(ITEM_MOON_TEAR) == ITEM_NONE) {
+                                gSaveContext.save.inventory.items[slot] = ITEM_MOON_TEAR;
+                            } else if ((INV_CONTENT(ITEM_MOON_TEAR) >= ITEM_MOON_TEAR) &&
+                                       (INV_CONTENT(ITEM_MOON_TEAR) <= ITEM_DEED_MOUNTAIN)) {
+                                gSaveContext.save.inventory.items[slot] = INV_CONTENT(ITEM_MOON_TEAR) + 1;
+                            }
+                        } else if (CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                            if (INV_CONTENT(ITEM_MOON_TEAR) == ITEM_NONE) {
+                                gSaveContext.save.inventory.items[slot] = ITEM_DEED_OCEAN;
+                            } else if ((INV_CONTENT(ITEM_MOON_TEAR) >= ITEM_DEED_LAND) &&
+                                       (INV_CONTENT(ITEM_MOON_TEAR) <= ITEM_DEED_OCEAN)) {
+                                gSaveContext.save.inventory.items[slot] = INV_CONTENT(ITEM_MOON_TEAR) - 1;
+                            }
+                        }
+                    } else if (slot == SLOT_TRADE_KEY_MAMA) {
+                        if (CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                            if (INV_CONTENT(ITEM_ROOM_KEY) == ITEM_NONE) {
+                                gSaveContext.save.inventory.items[slot] = ITEM_ROOM_KEY;
+                            } else if ((INV_CONTENT(ITEM_ROOM_KEY) >= ITEM_ROOM_KEY) &&
+                                       (INV_CONTENT(ITEM_ROOM_KEY) <= ITEM_ROOM_KEY)) {
+                                gSaveContext.save.inventory.items[slot] = INV_CONTENT(ITEM_ROOM_KEY) + 1;
+                            }
+                        } else if (CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                            if (INV_CONTENT(ITEM_ROOM_KEY) == ITEM_NONE) {
+                                gSaveContext.save.inventory.items[slot] = ITEM_LETTER_MAMA;
+                            } else if ((INV_CONTENT(ITEM_ROOM_KEY) >= ITEM_LETTER_MAMA) &&
+                                       (INV_CONTENT(ITEM_ROOM_KEY) <= ITEM_LETTER_MAMA)) {
+                                gSaveContext.save.inventory.items[slot] = INV_CONTENT(ITEM_ROOM_KEY) - 1;
+                            }
+                        }
+                    } else if (CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                        if (INV_CONTENT(ITEM_LETTER_TO_KAFEI) == ITEM_NONE) {
+                            gSaveContext.save.inventory.items[slot] = ITEM_LETTER_TO_KAFEI;
+                        } else if ((INV_CONTENT(ITEM_LETTER_TO_KAFEI) >= ITEM_LETTER_TO_KAFEI) &&
+                                   (INV_CONTENT(ITEM_LETTER_TO_KAFEI) <= ITEM_LETTER_TO_KAFEI)) {
```

Comment:
errrr, isn't this the same as `INV_CONTENT(ITEM_LETTER_TO_KAFEI) == ITEM_LETTER_TO_KAFEI` or am I missing something?

Reply:
That's correct... great programming here (yes, same thing with `ITEM_PENDANT_MEMORIES`)